ui/components/multichain/connect-accounts-modal/connect-accounts-modal.tsx

```diff
@@ -13,7 +14,9 @@ export const ConnectAccountsModal = ({
   activeTabOrigin: string;
 }) => {
   const accounts = useSelector((state) =>
-    getUnconnectedAccounts(state, activeTabOrigin),
+    getUnconnectedAccounts(state, activeTabOrigin).filter(
```

Comment:
I feel like we should explain why we are filtering them out (you can update the wording of my suggestion, it was just to give an example):

```suggestion
    // We only consider EVM accounts.
    // Connections with non-EVM accounts (Bitcoin only for now) are used implicitly and handled by the Bitcoin Snap itself.
    getUnconnectedAccounts(state, activeTabOrigin).filter(
```